test/docTest.js.html

```diff
@@ -499,19 +499,37 @@
           it('should broadcast event to all listeners', function() {
             var counter = 0;
             var data = 1;
-            $('#first-broadcast-listener').on('say hello to my little friend', function(event) {
+            $('#first-broadcast-listener').on('say-hello-to-my-little-friend', function(event) {
               counter += event.detail;
             });
-            $('#second-broadcast-listener').on('say hello to my little friend', function(event) {
+            $('#second-broadcast-listener').on('say-hello-to-my-little-friend', function(event) {
               counter += (event.detail * 2);
             });
-            $('#third-broadcast-listener').on('say hello to my little friend', function(event) {
+            $('#third-broadcast-listener').on('say-hello-to-my-little-friend', function(event) {
               counter += (event.detail * 3);
             });
-            $.broadcast('say hello to my little friend', data);
+            $.broadcast('say-hello-to-my-little-friend', data);
             assert.equal(counter, 6);
           });
 
+          it('should add multiple event listeners using one method', function() {
+              var counter = 0;
```

Comment:
It seems that this block is indented one level more than it should be.

@luiz

Reply:
🤖